app/js/track_events.js

```diff
@@ -0,0 +1,42 @@
+$(function() {
+  'use strict';
+
+  trackMenuClick();
+
+  analytics.trackForm($('.Nav form'), 'Subscribed Newsletter in the menu', getFormProperties);
+  analytics.trackForm($('.Newsletter form'), 'Subscribed Newsletter in the main content', getFormProperties);
+
+  $('[data-track-link]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLink'));
+  });
+
+  $('[data-track-link-mailto]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLinkMailto'), getMailFromMailTo);
+  });
+
+  function trackMenuClick() {
+    $('#BurgerBtn').on('click', function(event) {
+      var $btn = $(event.target);
+      if ($btn.hasClass('BurgerBtn--open')) {
+        analytics.track('Opened main menu');
+      } else {
+        analytics.track('Closed main menu');
+      }
```

Comment:
I see, you are right. I can't be sure when one will run.
Two suggestions:
1. Trigger an event when the class changes, and listen to it here then track the event
2. Create an object with a function for tracking the event (much like an angular service) attaching it to the `window`, then access it in nav.js. i.e. no listeners, you'd be explicitly requesting an event to be tracked

I'd go with 2., because it might be useful for future scenarios and it's easier to maintain